bbb-learning-dashboard/src/App.js

```diff
@@ -555,6 +601,20 @@ class App extends React.Component {
               </div>
             </div>
           </TabPanelUnstyled>
+          <TabPanelUnstyled value={4}>
+            <h2 className="block my-2 pr-2 text-xl font-semibold">
+              <FormattedMessage id="app.learningDashboard.pluginsTable" defaultMessage="H5P" />
```

Comment:
```suggestion
              <FormattedMessage id="app.learningDashboard.pluginsTable.title" defaultMessage="Plugins" />
```

Reply:
My bad, that should be the card title, and since it is generic for any generic data, I don't think we should even use the formatted message